pkg/skaffold/deploy/kpt.go

```diff
@@ -270,6 +280,45 @@ func (k *KptDeployer) kptFnRun(ctx context.Context) (deploy.ManifestList, error)
 	return manifests, nil
 }
 
+// ExcludeKptFn adds an annotation "config.kubernetes.io/local-config: 'true'" to kpt function.
+// This will exclude kpt functions from deployed to the cluster in kpt live apply.
+func (k *KptDeployer) ExcludeKptFn(manifest deploy.ManifestList) (deploy.ManifestList, error) {
```

Comment:
nit: does this func need to be exported?

Reply:
done.